research/src/components/file-anatomy.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+import './anatomy.css'
+
+const FileAnatomy = () => {
+  return (
+    <div class="component-anatomy-wrapper">
+      <input type="checkbox" id="show-slots" />
+      <label for="show-slots"> Show slots</label>
+      <div class="component-anatomy">
+        <host name="input">
```

Comment:
Can you file an issue about the naming of this? I think we should discuss having this simply be represented by `<file>` rather than input. This won't block the PR however since the current anatomy has been approved.

Reply:
Done! https://github.com/WICG/open-ui/issues/180